website_analytics_matomo/models/website.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2015 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields
+
+
+class Website(models.Model):
+    _inherit = 'website'
+
+    has_matomo_analytics = fields.Boolean("Matomo Analytics")
```

Comment:
for renamed fields, please include a migration script like https://github.com/OCA/e-commerce/blob/13.0/website_snippet_product_category/migrations/13.0.2.0.0/pre-migration.py, cf https://oca.github.io/OpenUpgrade/API.html#openupgradelib.openupgrade.rename_fields